File: app/src/main/java/org/lineageos/setupwizard/user/ScreenLockActivity.java
/*
 * SPDX-FileCopyrightText: 2016 The CyanogenMod Project
 * SPDX-FileCopyrightText: The LineageOS Project
 * SPDX-License-Identifier: Apache-2.0
 */

package org.lineageos.setupwizard.user;

import android.content.Intent;

import org.lineageos.setupwizard.base.SubBaseActivity;

public class ScreenLockActivity extends SubBaseActivity {

    private static final String ACTION_SETUP_LOCKSCREEN = "com.android.settings.SETUP_LOCK_SCREEN";

    @Override
    protected void onStartSubactivity() {
        Intent intent = new Intent(ACTION_SETUP_LOCKSCREEN);
        startSubactivity(intent);
    }
}

File: app/src/main/java/org/lineageos/setupwizard/user/ScreenLockActivity.kt
/*
 * SPDX-FileCopyrightText: 2016 The CyanogenMod Project
 * SPDX-FileCopyrightText: The LineageOS Project
 * SPDX-License-Identifier: Apache-2.0
 */

package org.lineageos.setupwizard.user

import android.content.Intent
import org.lineageos.setupwizard.base.SubBaseActivity

class ScreenLockActivity : SubBaseActivity() {

    override fun onStartSubactivity() {
        startSubactivity(Intent(ACTION_SETUP_LOCKSCREEN))
    }

    companion object {
        private const val ACTION_SETUP_LOCKSCREEN = "com.android.settings.SETUP_LOCK_SCREEN"
    }
}
